l10n/sleha/de-de/xml/article_geo_clustering.xml

```diff
@@ -0,0 +1,668 @@
+     </para>
+   </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Vermittler</term>
+    <listitem>
+     <para><remark>taroth 2017-12-29: FATE#322100 - move to Geo Guide? (see
+     c#6)</remark>
+      Sie haben einen dritten Rechner installiert, der nicht Teil eines vorhandenen Clusters ist und als Vermittler verwendet werden kann.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+
+  <para>
+   Detaillierte Informationen zu den einzelnen Anforderungen finden Sie auch in <xref linkend="sec-ha-geo-quick-req"/>.
+  </para>
+ </sect1>
+ <sect1 xml:id="sec-ha-geo-quick-req">
+  <title>Anforderungen</title>
+   <para>
+    Alle Rechner (Clusterknoten und Vermittler), die Teil des Clusters sind, müssen mindestens über folgende Module und Erweiterungen verfügen:
+   </para>
+
+<itemizedlist>
+   <listitem>
+    <para>Basesystem Module <phrase role="productnumber"><phrase os="sles">15 SP4</phrase></phrase></para>
+   </listitem>
+   <listitem>
+    <para>Server Applications Module <phrase role="productnumber"><phrase os="sles">15 SP4</phrase></phrase></para>
+   </listitem>
+   <listitem>
+    <para><phrase role="productname"><phrase os="sles">SUSE Linux Enterprise High Availability Extension</phrase></phrase> <phrase role="productnumber"><phrase os="sles">15 SP4</phrase></phrase></para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   Wählen Sie bei der Installation der Rechner <literal>HA GEO-Knoten</literal> als <systemitem>system role</systemitem>. Dadurch wird ein Minimalsystem installiert, bei dem die Pakete aus dem Schema <literal>Geo-Clustering für Hochverfügbarkeit (ha_geo)</literal> standardmäßig installiert werden.
+  </para>
+  <itemizedlist>
+   <title>Netzwerkanforderungen</title>
+   <listitem>
+   <para>
+    Auf die virtuellen IPs, die für die einzelnen Cluster-Sites verwendet werden, muss ein Zugriff über den GeoCluster möglich sein.
+   </para>
+  </listitem>
+   <listitem>
+   <para>
+     Die Sites müssen pro booth-Instanz über einen UDP- und einen TCP-Port erreichbar sein. Folglich müssen Firewalls oder IPSec-Tunnel zwischen ihnen entsprechend konfiguriert sein.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Weitere Entscheidungen bezüglich der Einrichtung machen es möglicherweise erforderlich, weitere Ports zu öffnen (z. B. für DRBD oder Datenbankreplikation).
+    </para>
+   </listitem>
+  </itemizedlist>
+
+  <itemizedlist>
+  <title>Sonstige Anforderungen und Empfehlungen</title>
+  <listitem>
+   <para>
+    Alle Cluster-Knoten auf allen Sites sollten mit einem NTP-Server außerhalb des Clusters synchronisiert werden. Sie finden weitere Informationen hierzu im<link xlink:href="https://documentation.suse.com/sles-15/html/SLES-all/cha-ntp.html"><citetitle>Administrationshandbuch</citetitle> für SUSE Linux Enterprise Server <phrase role="productnumber"><phrase os="sles">15 SP4</phrase></phrase></link>.
```

Comment:
missing space here: 'im<link...'

Reply:
Fix now in XML on my local cloned repo and pushed to 'Vistatec/localisation checkin sleha15sp4' branch in repo.

Reply:
Updated in TM.
![image](https://user-images.githubusercontent.com/68334239/182835183-ac4f3dee-3b47-4d02-a077-8a5097013d71.png)